### ISSUE TITLE
a question about calculating the loss in pretrainning

### ISSUE THREAD

**Issue by huzongxiang** [snap-stanford/pretrain-gnns#53]
# set up models, one for pre-training and one for context embeddings
model = GNN(args.num_layer, args.emb_dim, JK = args.JK, drop_ratio = args.dropout_ratio, gnn_type = args.gnn_type).to(device)
linear_pred_atoms = torch.nn.Linear(args.emb_dim, 119).to(device)
linear_pred_bonds = torch.nn.Linear(args.emb_dim, 4).to(device)

node_rep = model(batch.x, batch.edge_index, batch.edge_attr)
# loss for nodes
criterion = nn.CrossEntropyLoss()
pred_node = linear_pred_atoms(node_rep[batch.masked_atom_indices])
loss = criterion(pred_node.double(), batch.mask_node_label[:,0])

# question
Hello,
        I notice that you calculate the "CrossEntropyLoss" loss using an FFN layer "linear_pred_atoms" without softmax in your code pretrain_masking.py. I wonder why it's okay to do that?
